src/status_im2/common/standard_auth/view.cljs

```diff
@@ -0,0 +1,174 @@
+(ns status-im2.common.standard-auth.view
```

Comment:
nit: I think the name space for this ui should be `status-im2.common.standard-auth.keycard.view`
same for style

Reply:
@J-Son89 Sure, updated it.